app/src/main/java/com/team41/boromi/MainActivity.java

```diff
@@ -14,15 +15,18 @@
   @Inject
   AuthenticationController authController;
 
-  private FragmentManager manager = null;
   FragmentTransaction ft;
+  private FragmentManager manager = null;
 
   @Override
   protected void onCreate(Bundle savedInstanceState) {
     super.onCreate(savedInstanceState);
     setContentView(R.layout.activity_main);
     ((BoromiApp) getApplicationContext()).appComponent.getAuthenticationComponent().inject(this);
 
+    // TODO REMOVE THIS SO IT DOESN'T REDIRECT TO TEST ACTIVITY
+    startActivity(new Intent(this, TestActivity.class));
+
```

Comment:
I think should remove if merge to development

Reply:
smd lol jk kinda true